laget.Azure.ServiceBus/Topic/TopicSender.cs

```diff
@@ -22,21 +24,24 @@ public TopicSender(string connectionString, TopicOptions options)
             _client = new TopicClient(connectionString, options.TopicName, options.RetryPolicy);
         }
 
-
         public async Task SendAsync(IMessage message)
         {
-            var json = message.Serialize();
-            var bytes = Encoding.UTF8.GetBytes(json);
+            await _client.SendAsync(message.ToServicebusMessage());
+        }
 
-            await _client.SendAsync(new Microsoft.Azure.ServiceBus.Message(bytes));
+        public async Task SendAsync(IList<IMessage> messageList)
```

Comment:
Eftersom messageList bara loopas skulle man kunna köra på IEnumerable, men det är långt ifrån nåt måste att ändra.

Reply:
bra där! tänkte nog på fel lista när jag la in IList istället för IEnumerable. Fixat!